File: JavaBasic/167_rotate_left.java
// 167	167_rotate_left.java	Rotate an array to the left by one position.
import java.util.Arrays;
public class _167_rotate_left {
    public static void main(String[] args) {
        int[] arr = {1,2,3,4,5};
        System.out.println("Original: " + Arrays.toString(arr));
        int first = arr[0];
        System.arraycopy(arr, 1, arr, 0, arr.length-1);
        arr[arr.length-1] = first;
        System.out.println("Rotated left: " + Arrays.toString(arr));
    }
}

File: JSBASIC/167_rotate_left.js
// 167	167_rotate_left.js	Rotate an array to the left by one position.

let arr = [1, 2, 3, 4, 5];
console.log("Original:", arr);
let first = arr.shift();
arr.push(first);
console.log("Rotated left:", arr);
